JiraPS/Public/Find-JiraFilter.ps1

```diff
@@ -0,0 +1,89 @@
+function Find-JiraFilter {
+    # .ExternalHelp ..\JiraPS-help.xml
+    [CmdletBinding( SupportsPaging )]
+    param(
+        [Parameter(ValueFromPipeline,ValueFromPipelineByPropertyName)]
+        [string[]]$Name,
+
+        [Parameter(ValueFromPipelineByPropertyName)]
+        [string]$AccountId,
+
```

Comment:
I am missing the `owner`.
Which would be a different ParameterSet than `AccountId`

Reply:
Implemented ByAccountId and ByOwner parameter sets.